### DISCUSSION TITLE
How to UnAssign a Device from the Virtualization Clusters

### DISCUSSION THREAD

**Discussion by haristku** [netbox-community/netbox#14584] [Q&A] [answered]
I once Assign a Device to the Virtualization Clusters, and I cannot find a way to UnAssign it (undo Assign Device to Clusters)
please help..

**Comment by candlerb** [comment]
Netbox version?

In v3.6.7, it's as follows:
* Go to Virtualization > Clusters, click on an individual cluster (`/virtualization/clusters/1/`)
* Click the Devices tab (`/virtualization/clusters/1/devices/`)
* ~~Next to the device in question, click the dropdown next to the pencil icon, then Delete~~ EDIT: Check the selection box on the left-hand side next to the device(s) you want to remove, then the red "Remove Selected" button at the bottom

**Reply by haristku** [reply]
I'm using v3.6.7

> Next to the device in question, click the dropdown next to the pencil icon, then Delete (alternatively: check the selection box on the left-hand side, then the red "Remove Selected" button at the bottom)

that will removed the actual physical device ~~`/virtualization/clusters/1/`~~ (typo-edited: i meant `/dcim/devices/xx/`) permanently, what i want is just: take it out from the clusters (not to delete the device)

i did what you suggest, but then ~~`/virtualization/clusters/1/`~~ (typo-edited: i meant `/dcim/devices/xx/`) will no longer exists.

**Reply by candlerb** [reply, marked as answer]
> that will removed the actual physical device /virtualization/clusters/1/ permanently

`/virtualization/clusters/1/` is not a device, it's a cluster.

However, I'm sorry: I've checked, the pencil icon will indeed delete the device, as it takes you to `/dcim/devices/1/delete/`. But it first gives you a confirmation page saying are you sure you want to delete the device, so you can say no.

Using the checkbox to the left, and "remove selected", works. It takes you to `/virtualization/clusters/1/devices/remove/`. Here's the confirmation:

![image](https://github.com/netbox-community/netbox/assets/44789/72c5894c-f1ab-4e3d-a4ea-63a999dc88a2)

I've edited my answer above.

**Reply by haristku** [reply]
> /virtualization/clusters/1/ is not a device, it's a cluster.

ooo.. yes.. sorry.. my bad... i meant `/dcim/devices/xx/` will be removed... (edited)

> Using the checkbox to the left, and "remove selected", works. It takes you to /virtualization/clusters/1/devices/remove/. Here's the confirmation:

this works like charm....
I thought that action was the same, but it turned out to be different. 😄 

tysm..
💯
